src/modules/localization/domain/kalman_filter/Dynamics.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DYNAMICS_H
+#define DYNAMICS_H
```

Comment:
Specify the library name in guard (Good practice)
```suggestion
#ifndef KALMAN_FIKTER__DYNAMICS_H
#define KALMAN_FILTER__DYNAMICS_H
```